### PR TITLE
fix: create tpw table if doesn't exist

### LINKED ISSUE #227: Invalid grant / bigquery gcloud auth
I'm trying to get authenticated to update the bigquery but running into some challenges.

Readme indicates using gcp-credentials but that filename isn't supported anymore. But.. even then.. I'm not sure this approach is preferred.

@Nigui @kporal  what approach did yall use most recently?

I'm getting the farthest by creating a new KEY with my service account.. downloading that json. then

```sh
GOOGLE_APPLICATION_CREDENTIALS=thatseviceaccountkeyfile.json  node bin/automated-update.js
```

But still facing some permission challenges. I want to figure out the supported path so we can update the documentation.

### PR DESCRIPTION
**Pull request by Nigui**
This MR fixes broken pipeline when destination temporary table (`third_party_web`) doesn't exist. Script will check if exists then create with a schema if it doesn't. Fix #227 

It also bumps to bigquery latest (v7.9.0) to fix an issue when we target a dataset in a different project than the one locally configured.

It also move project to yarn v4 and nodejs v22 to fix an issue I had installing project on my macos machine (issue with canvas dependency - used to generate chart in readme).

This MR can be split if needed.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/.yarn/**            linguist-vendored
```
Comment (patrickhulce) [inline .gitattributes:1]: why check these in? shouldn't we just git ignore `.yarn`?

**Review by patrickhulce — COMMENTED** [review]
awesome thank you so much @Nigui 🎉

File: .gitattributes (hunk 1 above)
Comment (Nigui) [reply .gitattributes:1]: that's the file generated by yarn init. I think it's safe to keep it as is

Comment (patrickhulce) [reply .gitattributes:1]: yarn modern is super opinionated about determinism in a way that I'm not sure aligns with our needs here. I'd much prefer to not check in `.yarn/` and rely on `yarn.lock` being sufficient

File: .gitignore [hunk 2]
```diff
@@ -9,3 +9,6 @@ gcp-credentials.json
 *-subset.js
 *-subset.d.ts
 *.generated.sql
+
+
+.yarn
```
Comment (patrickhulce) [inline .gitignore:14]: ty ❤️

**Review by patrickhulce — APPROVED** [review]

**Pull request merged**